runtime/moonbase/src/lib.rs

```diff
@@ -1659,6 +1659,7 @@ impl pallet_proxy_genesis_companion::Config for Runtime {
 parameter_types! {
 	// Tells `pallet_base_fee` whether to calculate a new BaseFee `on_finalize` or not.
 	pub IsActive: bool = false;
+	pub DefaultBaseFeePerGas: U256 = (1 * currency::GIGAWEI * currency::SUPPLY_FACTOR).into();
```

Comment:
It might make sense to define this in `currency` since it's also used for `min_gas_price()`. But maybe not since this will change soon (?)